hud-pages/ahud-cc.md

```diff
@@ -5,26 +5,26 @@
 
 ### Modern update for Ahud
 
-- Support for more gamemodes (Payload, Bosses, Player destruction, Robot destruction,...)
-- HUD Customizations Menu (Scoreboard, Ubercharge meter, Streamer mode, Chat,...)
-- Community servers features (Play categories, special buttons)
-- Live improving features (Custom binds in option, Wiki for adding custom [Crosshair](https://github.com/jakadak/ahud-cc/wiki/Crosshair) & [Hitmarker](https://github.com/jakadak/ahud-cc/wiki/Hitmarker).
-- Live improving mods (No-smoke exposions, Consistent kill icons, Closed captions, Extended fonts)
-- 16:9, 16:10, and 4:3 support
+- Support for more gamemodes (Payload, Bosses, Player destruction, Robot destruction,...)  
+- HUD Customizations Menu (Scoreboard, Ubercharge meter, Streamer mode, Chat,...)  
+- Community servers features (Play categories, special buttons)  
+- Live improving features (Custom binds in option, Wiki for adding custom [Crosshair](https://github.com/jakadak/ahud-cc/wiki/Crosshair) & [Hitmarker](https://github.com/jakadak/ahud-cc/wiki/Hitmarker).  
+- Live improving mods (No-smoke exposions, Consistent kill icons, Closed captions, Extended fonts)  
+- 16:9, 16:10, and 4:3 support  
 
 Have a lovely day, au revoir!  
 ---
```

Comment:
```suggestion
# Have a lovely day, au revoir!
```

If I'm understanding the error correctly, this is what it wants you to do instead.